apps/opik-backend/src/main/java/com/comet/opik/domain/TextStreamer.java

```diff
@@ -0,0 +1,56 @@
+package com.comet.opik.domain;
+
+import com.comet.opik.utils.JsonUtils;
+import io.dropwizard.jersey.errors.ErrorMessage;
+import jakarta.inject.Singleton;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.glassfish.jersey.server.ChunkedOutput;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.concurrent.TimeoutException;
+
+@Singleton
+@Slf4j
+public class TextStreamer {
+
+    public ChunkedOutput<String> getOutputStream(@NonNull Flux<?> flux) {
+        var outputStream = new ChunkedOutput<String>(String.class, "\n");
```

Comment:
During our Hackathon, I had an issue streaming text when there were code blocks with new lines.  Just a heads up, this may be an issue.

Reply:
Your right here. The `chunked` directive for `Transfer-Encoding` expected Windows format `\r\n`. I'm too biased for Linux, so I choose `\n`, but I'm going to change it in the follow-up PR:
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding

This is fine to go in the meantime and it's just an endpoint skeleton.

Reply:
Done here: https://github.com/comet-ml/opik/pull/890